lib/core/base-provider.js

```diff
@@ -1,5 +1,5 @@
 const path = require("path");
-const ncConfig = require(path.join(process.cwd(), ".nc.config"));
+const ncConfig = require(path.resolve(process.cwd(), ".nc.config"));
```

Comment:
@rajitha1998 Can you do a manual test here, unit test has covered, just in case since this relates to core functionality

Reply:
Changing it to `require(path.resolve(process.cwd(), ".nc.config"));` from `require(path.join(process.cwd(), ".nc.config"));` does not fix the problem which is mentioned in #31. We might have to look for another solution

Reply:
Got it, let's fix the tests first then

Reply:
I fixed the tests :) @rajikaimal